packages/block-editor/src/components/block-inspector/README.md

```diff
@@ -0,0 +1,24 @@
+# Block Inspector
+
+The Block inspector is one of the panels on the right sidebar of the editor; it is mainly used to view and modify the settings of the selected block.
```

Comment:
I wonder if really need to mention "right sidebar" because it's a generic component that can be rendered anywhere. In the post editor, it happens to be rendered there.

Reply:
Yeah , makes sense. my initial description was how it is displayed in the editor.
Just edited.